cart/application/cartService.go

```diff
@@ -337,10 +340,36 @@ func (cs *CartService) UpdateItemSourceID(ctx context.Context, session *web.Sess
 	}
 
 	itemUpdate := cartDomain.ItemUpdateCommand{
-		SourceID: &sourceID,
+		SourceID:     &sourceID,
+		ItemID:       itemID,
+		DeliveryCode: deliveryCode,
 	}
 
-	cart, defers, err = behaviour.UpdateItem(ctx, cart, itemID, deliveryCode, itemUpdate)
+	cart, defers, err = behaviour.UpdateItem(ctx, cart, itemUpdate)
+	if err != nil {
+		cs.handleCartNotFound(session, err)
+		cs.logger.WithContext(ctx).WithField("subCategory", "UpdateItemSourceId").Error(err)
+
+		return err
+	}
+
+	return nil
+}
+
+// UpdateItemsSourceID updates multiple item source ids
+func (cs *CartService) UpdateItemsSourceID(ctx context.Context, session *web.Session, updateCommands cartDomain.ItemUpdateCommands) error {
```

Comment:
The function name is misleading since you could also update the qty with this call depending on the content of the `updateCommands`.. Maybe we should go for `UpdateItems`

Reply:
Check for mandatory fields om ItemUpdateCommands (ItemID, deliveryCode) could potentially be added here